docroot/themes/humsci/humsci_basic/src/js/shared/linked-cards/linked-cards.js

```diff
@@ -1,11 +1,19 @@
-function verticalCard() {
+function linkedCards() {
```

Comment:
No need to use a named function here. Please switch to using [a closure](https://www.drupal.org/docs/develop/standards/javascript-coding-standards/javascript-coding-standards#strict)